### PR TITLE
Adds plotutil example image width/height unit

### PR DESCRIPTION
**Pull request by zyluo**
Please take a look.

### CODE REVIEW

**Review by sbinet — DISMISSED** [review]
LGTM

File: plotutil/main.go [hunk 1]
```diff
@@ -35,37 +36,37 @@ func main() {
 }
 
 func drawEps(name string, mkplot func() *plot.Plot) {
-	if err := mkplot().Save(4, 4, name+".eps"); err != nil {
+	if err := mkplot().Save(4*vg.Inch, 4*vg.Inch, name+".eps"); err != nil {
```
Comment (kortschak) [inline plotutil/main.go:39]: Since these are being changed, `10*vg.Centimeter`.

**Review by kortschak — CHANGES_REQUESTED** [review]
@sbinet Do you know why the test expectations are no longer matching?

File: plotutil/main.go (hunk 1 above)
Comment (zyluo) [reply plotutil/main.go]: done

**Review by sbinet — DISMISSED** [review]
LGTM

(could you please add yourself to the `AUTHORS` and/or `CONTRIBUTORS` files of `gonum/gonum` ? thanks)

**Comment by sbinet** [comment]
@kortschak ah, I didn't notice.
I guess another change in the X/wayland stack that ripples on `vg/vggio`...

**Comment by zyluo** [comment]
I guess this patch is gated by vggio?

**Comment by kortschak** [comment]
If you rebase this onto master, it should pass.

**Comment by zyluo** [comment]
rebased to master

**Review by kortschak — APPROVED** [review]
Thanks

**Pull request merged**